Model/src/BoardPackage/FreeParkingSquare.java

```diff
@@ -1,7 +1,20 @@
 package BoardPackage;
 
-/**
- * Created by Dylan Kernohan on 2/10/18.
- */
-public class FreeParkingSquare {
+/*********************************************************************
+ * The FreeParkingSquare is a board square on the Monopoly game board
+ * where when you land on it nothing happens. This class exist however
+ * for the possiblity of adding functionality for this board square
+ * later.

```

Comment:
Add the version and the author to the class description.

Reply:
Agreed

Reply:
^

Reply:
^